src/pages/Landing.js

```diff
@@ -56,14 +89,41 @@ const Landing = () => {
     };
 
     const loginHandler = () => {
-        login(loginInfo.email, loginInfo.password)
-            .then((value) => {
-                console.log("val", value);
-                auth.acceptLogin(loginInfo.email, value.token);
-            })
-            .catch((err) => {
-                console.log(err);
-            });
+        if ("email" in loginInfo && "password" in loginInfo) {
+            loginToast.current = toast.loading("Logging In...")
+            login(loginInfo.email, loginInfo.password)
+                .then((value) => {
```

Comment:
Try to condense into try catch and await the login call instead of using .then callback to lessen number of lines of code